contrib/actions/generate-actions.py

```diff
@@ -25,13 +25,19 @@
     steps:
       - uses: actions/checkout@v1
       - name: Install Dependencies
-        run: bash contrib/actions/install-dependencies.sh
+        run: sudo contrib/actions/install-dependencies.sh
       - name: Build
-        run: bash contrib/actions/run-build.sh {target_name}
+        run: contrib/actions/run-build.sh {target_name}
+      - name: Archive build logs
+        if: ${{{{ !cancelled() }}}}
+        uses: actions/upload-artifact@v1
+        with:
+          name: {target_name}-logs
```

Comment:
```suggestion
          name: {target_name}/logs
```

IMO improves readability

Reply:
Ah, I didn't know this was valid.

Reply:
```
Artifact name is not valid: ar71xx-tiny/logs. It cannot contain '\', '/', "', ':', '<', '>', '|', '*', and '?'
```
Options:
- `{target_name}.logs`
- `{target_name}@logs`
- `{target_name}+logs`
- `{target_name}_logs`
- `{target_name}~logs`
- ...

Also, it looks like they failed to escape `'` and/or `"` correctly in their error message ;)

Reply:
I would sick with `-` then

Reply:
I'm leaning towards `{target_name}_logs` or maybe `{target_name}--logs`. @mweinelt @rotanid ?

Reply:
I'm leaning towards anything but `-`. it needs to be visually different from `target-subtarget`.

Reply:
Go with `_` then